Documentation/help_files/cms_1500_help.php

```diff
@@ -0,0 +1,154 @@
+                
+                <p><?php echo xlt("The CMS 1500 form has 33 boxes and is divided into two sections");?>.&nbsp;<a href="https://www.cms.gov/Medicare/CMS-Forms/CMS-Forms/Downloads/CMS1500.pdf" target="_blank"><i id="show_hide" class="fa fa-eye fa-lg small" title=<?php echo xlt("Click to Show");?>></i></a>
+                
+                <p><?php echo xlt("Patient and Insured Informatiom - Boxes 1 - 13 and  Physician or Supplier Information - Boxes 14 - 33");?>.
+                
+                <p><?php echo xlt("The information provided in the New Encounter Form and Fee Sheet in openEMR is used to generate the paper claim as well as electronic claims");?>.
+                
+                <p><?php echo xlt("Some global parameters on the setup and printing of the generated claims can be set in Adminstration > Globals > Billing");?>.
+                
+                <p><i class="fa fa-exclamation-triangle oe-text-red" aria-hidden="true"></i> <strong><?php echo xlt("You need administrator privileges to perform the setup"); ?>.</strong>
+                
+                <p><?php echo xlt("The miscellaneous billing options for HCFA-1500 form is used to provide additional information for an individual claim");?>.
+                
+                <p><?php echo xlt("It is divide into two sections:");?>.
+                
+                <ul>
+                    <li><a href="#select_option"><?php echo xlt("Select Options for Current Encounter");?></a></li>
+                    <li><a href="#add_notes"><?php echo xlt("Additional Notes");?></a></li>
+                </ul>
+            </div>
+            <div class= "row" id="select_option">
+                <h4 class="oe-help-heading"><?php echo xlt("Select Options for Current Encounter"); ?><a href="#"><i class="fa fa-arrow-circle-up float-right oe-help-redirect" aria-hidden="true"></i></a></h4>
+                <p><?php echo xlt("Enter needed information to help process this particular claim");?>.
+                
+                <p><?php echo xlt("Checking any of the check boxes will result in a Yes on the CMS 1500 form");?>.
+                
+                <p><?php echo xlt("Box 10 - This information indicates whether the patient’s illness or injury is related to employment, auto accident, or other accident");?>.
+                
+                <p><?php echo xlt("Box 10 D  - Is used to submit the Early and Periodic Screening, Diagnosis and Treatment (EPSDT) Referral Code");?>.
+                
+                <p><?php echo xlt("Box 14 - Is Populated from the Encounter Screen as the Onset Date. It identifies the first date of onset of illness, the actual date of injury, or the LMP for pregnancy");?>.
+                
+                <p><?php echo xlt("Box 16 - Dates Patient Unable to Work in Current Occupation  - is the time span the patient is or was unable to work");?>.
```

Comment:
Ensure no double spacing in these translation constants (such as the line above)